apps/website/core-components/toggle/README.md

```diff
@@ -81,3 +81,123 @@ Toggle switches take up less space than an “on/off” radio button group and c
     </cds-toggle>
 </div>
 </DocIndent>
+
+::: component-section-level-one-title
+
+## Layouts
+
+:::
+
+::: component-section-level-one
+
+Toggle is available in vertical, vertical inline, horizontal, horizontal inline, compact layout. The align options for Toggle are useful for creating continuity with other UI elements that are related to the toggle selection. See more layout guidelines on the form page. [form page](/web-components/form/).
```

Comment:
`horizontal, horizontal inline, and compact layout` not `horizontal, horizontal inline, compact layout`